ci-operator/step-registry/baremetal/lab/pre/load-balancer/run/baremetal-lab-pre-load-balancer-run-commands.sh

```diff
@@ -98,12 +98,20 @@ podman kill --signal HUP "haproxy-$CLUSTER_NAME"
 echo "Gather the IP Address for the new interface"
 
 api_ip=$(nsenter -m -u -n -i -p -t "$(podman inspect -f '{{ .State.Pid }}' "haproxy-${CLUSTER_NAME}")" -n  \
-  /sbin/ip -o -4 a list eth1 | sed 's/.*inet \(.*\)\/[0-9]* brd.*$/\1/')
+  /sbin/ip -o -4 a list eth2 | sed 's/.*inet \(.*\)\/[0-9]* brd.*$/\1/')
```

Comment:
can you clarify why we are changing this to eth2?

Reply:
This is for testing purpose.